src/resources/parser/glb-parser.js

```diff
@@ -0,0 +1,1466 @@
+            //                 var numValues = numPoints * attribute.num_components();
+            //                 var values = new Float32Array(numValues);
+            //
+            //                 for (i = 0; i < numValues; i++) {
+            //                     values[i] = attributeData.GetValue(i);
+            //                 }
+            //
+            //                 decoderModule.destroy(attributeData);
+            //                 return values;
+            //             };
+            //
+            //             var dracoAttribs = extDraco.attributes;
+            //             if (dracoAttribs.hasOwnProperty('POSITION'))
+            //                 positions = extractAttribute(dracoAttribs.POSITION);
+            //             if (dracoAttribs.hasOwnProperty('NORMAL'))
+            //                 normals   = extractAttribute(dracoAttribs.NORMAL);
+            //             if (dracoAttribs.hasOwnProperty('TANGENT'))
+            //                 tangents  = extractAttribute(dracoAttribs.TANGENT);
+            //             if (dracoAttribs.hasOwnProperty('TEXCOORD_0'))
+            //                 texCoord0 = extractAttribute(dracoAttribs.TEXCOORD_0);
+            //             if (dracoAttribs.hasOwnProperty('TEXCOORD_1'))
+            //                 texCoord1 = extractAttribute(dracoAttribs.TEXCOORD_1);
+            //             if (dracoAttribs.hasOwnProperty('COLOR_0'))
+            //                 colors    = extractAttribute(dracoAttribs.COLOR_0);
+            //             if (dracoAttribs.hasOwnProperty('JOINTS_0'))
+            //                 joints    = extractAttribute(dracoAttribs.JOINTS_0);
+            //             if (dracoAttribs.hasOwnProperty('WEIGHTS_0'))
+            //                 weights   = extractAttribute(dracoAttribs.WEIGHTS_0);
+            //         }
+            //
+            //         if (geometryType == decoderModule.TRIANGULAR_MESH) {
+            //             var face = new decoderModule.DracoInt32Array();
+            //             indices = (numPoints > 65535) ? new Uint32Array(numFaces * 3) : new Uint16Array(numFaces * 3);
+            //             for (i = 0; i < numFaces; ++i) {
+            //                 decoder.GetFaceFromMesh(outputGeometry, i, face);
+            //                 indices[i * 3]     = face.GetValue(0);
+            //                 indices[i * 3 + 1] = face.GetValue(1);
+            //                 indices[i * 3 + 2] = face.GetValue(2);
+            //             }
+            //             decoderModule.destroy(face);
+            //         }
+            //
+            //         decoderModule.destroy(outputGeometry);
+            //         decoderModule.destroy(decoder);
+            //         decoderModule.destroy(buffer);
+            //     }
+            // }
+            // //
```

Comment:
I'd recommend just deleting all the Draco stuff for now. It can still be seen in the playcanvas-gltf repo if needed.

Reply:
done